src/components/landing/listing.js

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import Image from "next/image";
+import styles from "./listing.module.css";
+function Listing({ data }) {
+  return (
+    <>
+      <article className={styles.listing}>
+        <div className={styles.listing__image__container}>
+          <Image src="/image-placeholder.png" width={50} height={50} alt={`${data.name} avatar`} />
+        </div>
+        <div className={styles.listing__content}>
+          <h2 className={styles.listing__content__name}>{data.community_name}</h2>
+          <p className={styles.listing__content__manager}>{data.manager}</p>
```

Comment:
I think this should have a different styling from the `data.description` paragraph, just to improve UX and follow basic design principle

Reply:
fixed @annonymousauthority